### PR TITLE
Add Implemented algorithms section

### PR DESCRIPTION
**Pull request by alexsaaf**
Not 100% sure about how we want to format the info about the implemented algorithms but I figured it's a start.

### CODE REVIEW

**Comment by holmgr** [comment]
Looks pretty good to me. I was however thinking more in terms of a sort of table. Where we have the name, file, time/space-complexity and type (in-place, recursive, etc.). What do you think?

**Comment by alexsaaf** [comment]
Yeah, you're right. That will probably work better, especially when the number of implemented algorithms start to increase.

File: README.md [hunk 1]
```diff
@@ -58,3 +58,11 @@ Data structures
 - [] Persistant rope
 - [] Priority Queue (min or max heap)
 - [] Prefix tree
+
+## Implemented algorithms
+This list contains the algorithms implemented so far. 
+
+|   Algorithm name	| Description | File | Time complexity	| Space complexity  | Notes  |
+|---	|---  |---	|---	|---	|---	|
+|  Quick sort 	| Sorts a list in place | sorting.cpp 	| O(nlogn) (O(n^2) worst case)  	|   ?	|  In place |
```
Comment (holmgr) [inline README.md]: I think space complexity is O(n), otherwise it looks perfect!

**Review by holmgr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alexsaaf) [reply README.md]: Yeah you're right, adding that!

**Review by holmgr — APPROVED** [review]
Ready to merge :+1:

**Pull request merged**